external/storm-cassandra/src/main/java/org/apache/storm/cassandra/bolt/BaseCassandraBolt.java

```diff
@@ -92,6 +94,8 @@ public void prepare(Map stormConfig, TopologyContext topologyContext, OutputColl
         this.outputCollector = outputCollector;
         this.stormConfig = stormConfig;
         this.cassandraConfConfig = new CassandraConf(stormConfig);
+
+        if(configKey != null)
         this.client = clientProvider.getClient(this.stormConfig);
```

Comment:
Right, somehow the changes I see in my IDE and saved file in disk are different. I will update with the right sources. Thanks for raising this.